src/JuliusSweetland.OptiKey/UI/ValueConverters/IsValidImage.cs

```diff
@@ -0,0 +1,253 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Win32;
+
+namespace JuliusSweetland.OptiKey.UI.ValueConverters
+{
+    public class IsValidImage : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            string image = value.ToString();
+            if (String.IsNullOrEmpty(image))
```

Comment:
string. (I prefer using the keyword over the class - just a personal preference)